WC_Layers/Photon_TallytoVtk.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Oct  3 23:18:20 2024
+
+@author: Anupama Rajendra
+"""
+import openmc
+import matplotlib.pyplot as plt
+
+def read_statepoint(sp_filename, flux_spectrum_tally_id, mesh_number, summed_axes_energy_filter, energy_filter_index):
+    with openmc.StatePoint(sp_filename) as sp:
+        flux_spectrum = sp.get_tally(id=flux_spectrum_tally_id) 
+        mesh=sp.meshes[mesh_number]
+        # Return tally data condensed into 1 dimension per filter
+        tally_data_reshaped = flux_spectrum.get_reshaped_data(value='mean')
+        flux_sum_energy = tally_data_reshaped.sum(axis=summed_axes_energy_filter)   

```

Comment:
It's not clear to me what axes this data is being summed over.  The data is originally a flux spectrum of G groups for each of M mesh elements. I assumed that this sum was summing over G, to produce a total flux for each of the M mesh elements.  However, when you use this in the plot below, it seems like it may be summing over M to produce a single flux spectrum of G groups.  If so, then we need to make sure that the volume normalization is being done correctly. (Full disclosure: I'm not sure when/how volume normalization of tallies happens in OpenMC.  If this was MCNP-like, then this operation would be wrong because the results would already be volume normalized per mesh element, and summing would require a volume-weighting.)